docs/DeveloperGuide.adoc

```diff
@@ -46,7 +46,9 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* * *` |user |add a new person |
 |`* * *` |user |delete a person |remove entries that I no longer need
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* *` |user |list all the people in the address book |see everyone that is in the address book 
 |`* *` |user |hide <<private-contact-detail, private contact details>> by default |minimize chance of someone else seeing them by accident
+|`*` |user |find a person by contact number |locate a person and the details without having to look through the entire list
```

Comment:
In this user story, this is good. However, you can try to be more specific when you write user stories for your project

Reply:
Okay! Thank you!